docs/sphinx/index.rst

```diff
@@ -110,6 +110,8 @@ Uberenv has a few options that allow you to control how dependencies are built:
   ``-k``                  Ignore SSL Errors                                    **False**
   ``--install``           Fully install target, not just dependencies          **False**
   ``--run_tests``         Invoke tests during build and against install        **False**
+  ``--setup-only``        Only download and setup Spack                        **False**
+  ``--skip_setup``        Only install (using pre-setup Spack)                 **False**
```

Comment:
I think we should use either "-" or "_" in option names, for consistency and ease of use. I suggest "-".

Reply:
turns out in `uberenv.py` itself i had it with a "-" but in the docs it was "_", for `skip-setup`.

theres also `--run_tests` which uses an underscore. i would change it to use a hyphen but i think that would cause issues for users.